de_project/de_app/views.py

```diff
@@ -1,9 +1,15 @@
 from django.shortcuts import render
 
-# Create your views here.
+from .models import RecordHandler
 
 from django.http import HttpResponse
+from django.http import JsonResponse
 
 
 def index(request):
-    return HttpResponse("Hello, world!")
\ No newline at end of file
+    return HttpResponse("Hello, world!")
+
+def get_data_as_json_file(request, record_handler_name):
+    record_handler = RecordHandler.objects.get(name=record_handler_name)
```

Comment:
Make sure to handle the edge case where the provided record handler doesn't exist and return 404

Reply:
Good point! That is added.